tools/src/main/java/org/apache/rocketmq/tools/command/acl/GetAccessConfigSubCommand.java

```diff
@@ -52,10 +52,10 @@ public class GetAccessConfigSubCommand implements SubCommand {
     @Override public Options buildCommandlineOptions(Options options) {
         OptionGroup optionGroup = new OptionGroup();
 
-        Option opt = new Option("b", "brokerAddr", true, "query acl config version for which broker");
+        Option opt = new Option("b", "brokerAddr", true, "query acl config for which broker");
```

Comment:
Hi @meateggmilk ,

Please help us correct the description into "query acl config for specified cluster" to align with the -c option. Thx.

Reply:
ok, done it